src/pt/argosscan/src/eu/kanade/tachiyomi/extension/pt/argosscan/ArgosScan.kt

```diff
@@ -75,18 +71,19 @@ class ArgosScan : ParsedHttpSource() {
 
     // ============================ Search ======================================
 
-    override fun searchMangaRequest(page: Int, query: String, filters: FilterList): Request {
-        val url = baseUrl.toHttpUrl().newBuilder()
-            .addQueryParameter("s", query)
-            .build()
-        return GET(url, headers)
+    override fun fetchSearchManga(page: Int, query: String, filters: FilterList): Observable<MangasPage> {
+        return fetchPopularManga(page).map { (mangas, _) ->
```

Comment:
I belive this will not work of tachi sy, since `MangasPage` isn't a data class there, so it doesn't have `component` functions
```kt
val mangas = fetchPopularManga(page).mangas
...
```